src/main/java/racingcar/dto/PlayerDto.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.dto;
+
+import racingcar.domain.Name;
+import racingcar.domain.Position;
+
+public class PlayerDto {
+    private final Name name;
+    private final Position position;
```

Comment:
DTO 에서 도메인 VO 를 알고 있어야할지 의문이 생겼어요 🤔

Reply:
DTO 내부에 VO를 가지고 있는 이유는 값을 getter로 가져오게 될 경우 바로 사용할 수 있도록 하기 위해서였습니다.

DTO와 VO에는 차이점이 있는데
DTO는 데이터 접근 이외의 기능을 가지지않고
VO는 특정한 비즈니스 로직을 가질 수 있습니다.

지금 제가 만든 DTO에는 특정한 비즈니스 로직을 가진 VO를 필드로 갖고 있기때문에 데이터 접근 기능과 함께 특정한 비즈니스 로직을 수행할 수 있게 됐습니다.

그렇다면 DTO에서 VO를 알고있지 않아야 합니다.

하지만 DTO 내부에서는 특정한 비즈니스 로직을 수행하지 않는 타입(String, int, Integer, long, Long, ...)만을 사용해야 할지 궁금합니다. 🤔

이번 미션을 진행하면서 의도하지 않은 곳에서 값의 변경이 일어날 수도 있다는 것을 생각하지 않았다는 것도 알게 됐습니다. 👍
```suggestion
    private String name;
    private int position;
```

Reply:
> 하지만 DTO 내부에서는 특정한 비즈니스 로직을 수행하지 않는 타입(String, int, Integer, long, Long, ...)만을 사용해야 할지 궁금합니다. 🤔

꼭 그렇다! 라고는 할 수 없지만
DTO 는 비즈니스 로직을 포함하지 않는게 좋다~ 라는 관점에서 사용하다보면 원시 타입을 위주로 사용하게 되더라구요 😄 
또는 다른 DTO 를 의존할 수도 있구요

```java
class GameDto {
    private final ResultDto resultDto;
}
```